Sources/HummingbirdXCT/HBXCT.swift

```diff
@@ -48,19 +49,44 @@ struct HBXCTError: Error, Equatable {
     static var timeout: Self { .init(.timeout) }
 }
 
-/// Protocol for XCT framework.
-public protocol HBXCT {
-    /// Called to start testing of application
-    func start(application: HBApplication) throws
-    /// Called to stop testing of application
-    func stop(application: HBApplication)
+/// Protocol for client used by HBXCT
+public protocol HBXCTClientProtocol {
```

Comment:
Is it possible to make this conform to Sendable?

Reply:
Not easily. `HBXCTRouter.Client` holds an `HBResponder` which is not `Sendable`. Setting this to `Sendable` now would have major repercussions. I'll add a comment to revisit